monai/handlers/surface_distance.py

```diff
@@ -38,7 +38,8 @@ def __init__(
                 `seg_pred` and `seg_gt`. Defaults to ``False``.
             distance_metric: : [``"euclidean"``, ``"chessboard"``, ``"taxicab"``]
                 the metric used to compute surface distance. Defaults to ``"euclidean"``.
-            output_transform: transform the ignite.engine.state.output into [y_pred, y] pair.
+            output_transform: callable to construct the `[y_pred, y]` pair from `ignite.engine.state.output`, where
```

Comment:
Same suggestion as written in `ignite_metric.py`.

Reply:
updated in the latest commit.
Thanks.